src/stores/utils/web3.js

```diff
@@ -1,13 +1,30 @@
 import Web3 from 'web3'
 import { fromWei, toHex } from 'web3-utils'
 
+const updateTitle = (networkName = 'No chain specified') => {
```

Comment:
Can you consider to change the logic a bit?

```javascript
const updateTitle = (networkName = 'No chain specified') => {
  const defaultTitle = 'TokenBridge UI app'
  if (!process.env.REACT_APP_TITLE) {
    document.title = defaultTitle
  }
  else {
    const titleReplaceString = '%c'
    let appTitle = process.env.REACT_APP_TITLE || defaultTitle

    if (appTitle.indexOf(titleReplaceString) !== -1) {
      document.title = appTitle.replace(titleReplaceString, networkName)
    }
    else {
      document.title = appTitle
    }
  }
}
```

Reply:
Sure, looks simpler that way.